pkg/internal/tbtctest/tbtctest.go

```diff
@@ -20,6 +20,36 @@
 //   one input (a P2WSH deposit) was swept into a P2WPKH main UTXO.
 //   For reference see:
 //   https://live.blockcypher.com/btc-testnet/tx/9efc9d555233e12e06378a35a7b988d54f7043b5c3156adc79c7af0a0fd6f1a0
+//
```

Comment:
Could be worth noting all redemption test scenarios have a single P2WPKH input.

Reply:
See 378ff634